resources/restricted-secure-supply-chain-verification-config.yaml

```diff
@@ -0,0 +1,23 @@
+# Default Kubewarden verification config
+#
+# With this config, the only valid policies are those signed by Kubewarden
+# infrastructure.
+#
+# This config can be saved to its default location (for this OS) with:
+#   kwctl scaffold verification-config > /home/jvanz/.config/kubewarden/verification-config.yml
+#
+# Providing a config in the default location enables Sigstore verification.
+# See https://docs.kubewarden.io for more Sigstore verification options.
+---
+apiVersion: v1
```

Comment:
I think verification should not pass with this config unless I'm missing something here. I think it should be something like:

```
allOf:
  - kind: genericIssuer
    issuer: https://token.actions.githubusercontent.com
    subject:
      urlPrefix: https://github.com/kubewarden
 
```

Reply:
ignore this, I just realized this is the one that should fail :sweat_smile: